zerver/actions/message_send.py

```diff
@@ -484,6 +489,33 @@ def do_schedule_messages(send_message_requests: Sequence[SendMessageRequest]) ->
     return [scheduled_message.id for scheduled_message in scheduled_messages]
 
 
+def edit_scheduled_message(
+    scheduled_message_id: str, send_request: SendMessageRequest, sender: UserProfile
```

Comment:
I would make this expect an `int`; the views code calling this is usually responsible for parsing input and thus making sure it's passing us an integer for `scheduled_message_id`.

Reply:
Done.

Reply:
This further needs to be tweaked to use `json_validator=check_int` for parsing; I'll just do that change since the commit otherwise looks good.